cmd/crc/cmd/status.go

```diff
@@ -79,25 +79,41 @@ func runWatchStatus(writer io.Writer, client *daemonclient.Client, cacheDir stri
 		ramBar  *pb.ProgressBar
 	)
 
+	var err error
+	barPull = pb.NewPool()
 	isPullInit := false
-
-	err := client.SSEClient.Status(func(loadResult *types.ClusterLoadResult) {
+	err = barPull.Start()
```

Comment:
Do you need to start the pool right away? Can we wait we get to `if !isPullInit()` before calling `Start()` as it was done before?

Reply:
I made some test for that and we need to have `barPull.Start()`.
As simplification we could use `pb.StartPool()` instead of `pb.NewPool()` to remove that line